app/src/main/java/com/oracle/javafx/scenebuilder/app/DocumentWatchingController.java

```diff
@@ -58,49 +58,31 @@ public class DocumentWatchingController implements FileWatcher.Delegate {
     private final SceneStyleSheetMenuController sceneStyleSheetMenuController;
     private final FileWatcher fileWatcher 
             = new FileWatcher(2000 /* ms */, this, DocumentWindowController.class.getSimpleName());
-    
-    
+
     public DocumentWatchingController(DocumentWindowController documentWindowController) {
         this.documentWindowController = documentWindowController;
         this.editorController = documentWindowController.getEditorController();
         this.resourceController = documentWindowController.getResourceController();
         this.sceneStyleSheetMenuController = documentWindowController.getSceneStyleSheetMenuController();
         
-        this.editorController.sceneStyleSheetProperty().addListener(
-                (ChangeListener<ObservableList<File>>) (ov, t,
-                        t1) -> update());
+        this.editorController.sceneStyleSheetProperty().addListener((ov, t, t1) -> update());
```

Comment:
should we replace the change listener with an invalidation listener?

Reply:
Actually, not sure. Is there a semantic difference in this case? This is the sequence I am imagining:

1. the stylesheet property becomes invalid
2. we `update()` (and through `editorController.getSceneStyleSheets()` we effectively ask the property to re-compute itself)

Overall, I think it would behave in a way similar to the change listener (possibly in a different order compared to above). Though I'm not brave enough to make the switch without a proper test 😃

Reply:
There shouldn't be any semantic difference. I prefer to avoid change listeners if the `old` or `new` values of the property are not used.

We can keep it as it is for now.